branch-and-stage.sh

```diff
@@ -2,6 +2,24 @@
 
 source "${GITHUB_ACTION_PATH}/util.sh"
 
+pathsToCopy="$1"
```

Comment:
we should check first if `pathsToCopy` contains, if not let's skip the whole loop. what do you think?

Reply:
I added empty array by default so it will be skipped if not specified